docs/source/backends/llamacpp.md

```diff
@@ -101,10 +108,10 @@ The table below summarizes key options:
 | `--split-mode`                      | Split the model across multiple GPUs                                   |
 | `--defrag-threshold`                | Defragment the KV cache if holes/size > threshold                      |
 | `--numa`                            | Enable NUMA optimizations                                              |
-| `--use-mmap`                        | Use memory mapping for the model                                       |
+| `--disable-mmap`                    | Disable memory mapping for the model                                   |
```

Comment:
Your call on this, but I like this kind of stuff to be automated and checked by CI.

Reply:
Totally agree, I will look into automating it in a future PR.